reth/download-apply-snapshot.sh

```
@@ -32,12 +22,17 @@ fi
 readonly SNAPSHOT_BASE_URL_DEFAULT="https://snapshots.lisk.com"
 readonly SNAPSHOT_BASE_URL_ALTERNATE="https://s3.fr-par.scw.cloud/snapshots.lisk.com"
 
-# Automatically resolve SNAPSHOT_URL, if not specified
+# Automatically resolve SNAPSHOT_URL, if not specified. When SNAPSHOT_URL is
+# given explicitly (e.g. a Gelato URL), SNAPSHOT_TYPE is irrelevant.
 SNAPSHOT_URL="$SNAPSHOT_URL"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)